tests/test_resource.py

```diff
@@ -221,6 +221,18 @@ def test_method(self):
             "request_id": ANY,
         }
 
+    def test__get_name_from__name_parameter(self):
+        class XResource(Resource):
+            _name = "test"
+
+        assert XResource.get_name() == "test"
+
+    def test__get_name_from___name___parameter(self):
```

Comment:
Too many double underscores.

Reply:
the parametere is __name__ so in test we should add one more on both ends - that was my logic but 2 still should be sufficient

Reply:
Ok, now I get it but underscores in the name of the test means separation, so doing it this way, you decrease the readability 😉  What would you say for simple:
```python
def test__get_name__uses_class_name_attribute_when_custom_name_is_not_defined(self):
```

That is compatible with the pattern `test__tested_method__what_is_tested()` 😉